layouts/section/archive.html

```diff
@@ -0,0 +1,29 @@
+{{ define "main" }}
+<main class="container">
+  <section class="row section">
+    <div class="col-12 col-md-8 push-md-4">
+      {{ $paginator := .Paginate (where .Site.Pages "Type" "post") }}
+      {{ range $paginator.Pages.GroupByDate "2006"}}
```

Comment:
I think Hugo parses the "2006" and understands that to be the format for "year". It is in Hugo documentation's sample (probably because that's the year the feature was introduced).